src/compartment.c

```diff
@@ -120,8 +120,8 @@ comp_from_elf(char* filename)
         this_seg->prot_flags = (comp_phdr.p_flags & PF_R ? PROT_READ : 0) |
                                 (comp_phdr.p_flags & PF_W ? PROT_WRITE : 0) |
                                 (comp_phdr.p_flags & PF_X ? PROT_EXEC : 0);
-        printf("Added segment for header %lu:\n", i);
-        segmap_print(this_seg);
+        /*printf("Added segment for header %lu:\n", i);*/
```

Comment:
Commented code.

Reply:
Fixed in https://github.com/capablevms/CHERI-ELF-comp/pull/3/commits/00073d1d86c091d037fc9ba6e931856eda0333fd.